### DISCUSSION TITLE
while pushing docker image to ECR getting denied: Your authorization token has expired. Reauthenticate and try again.

### DISCUSSION THREAD

**Discussion by pcs1999** [learndevopsonline/learndevopsonline#1065] [Q&A]
please help here raghu 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/120551261/0bd7d473-7237-48fa-86f5-c22fd5692ea0)

**Comment by anandnandhu1** [comment]
can u help me this bro how to slove this error
